src/Sulu/Component/Webspace/Url/UrlProviderInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace\Url;
+
+use Sulu\Component\Webspace\Url;
+use Sulu\Component\Webspace\Webspace;
+
+/**
+ * Provides urls for given webspace.
+ */
+interface UrlProviderInterface
```

Comment:
How about calling this one `WebspaceUrlProvider`? Would make things clearer to me.